packages/babel-preset-expo/__tests__/index-test.js

```diff
@@ -72,3 +72,48 @@ import 'react-native-vector-icons';
   expect(code).toMatch(/"@expo\/vector-icons"/);
   expect(code).toMatchSnapshot();
 });
+
+it(`uses lazy - default (no option) matches option "null"`, () => {
```

Comment:
Wrap these new tests in:
```
describe('"lazy" option', () => {
  it (`defaults to null`, () => ... );
});
```

Jest will print out `"lazy" options defaults to null` in its test output for you.